catkin_tools/jobs/utils.py

```diff
@@ -25,6 +25,21 @@
 from catkin_tools.execution.events import ExecutionEvent
 
 
+class CommandMissing(Exception):
+    '''A required command is missing.'''
+
+    def __init__(self, name):
+        super(CommandMissing, self).__init__(
+                'Cannot find required tool `%s` on the PATH, is it installed?' % name)
```

Comment:
(A good motivation to do so might be eliminating the use of `sys.exit` within `build_isolated_workspace`, in favour of throwing other exceptions that resolve to the string errors messages. Doing this would make the function more cooperative as far as being wrapped or called from other tools.)